terraform/README.md

```diff
@@ -0,0 +1,60 @@
+# Terraform module for oidc-gatekeeper
```

Comment:
I think we 'd need to include compatibility due to changes in https://github.com/canonical/oidc-gatekeeper-operator/blob/track/ckf-1.9/metadata.yaml (see thread in channel)